server/mediapublic/tests/velruse_stub/__init__.py

```diff
@@ -0,0 +1,32 @@
+from wsgiref.simple_server import make_server
+from pyramid.config import Configurator
+from pyramid.view import view_config
+from pyramid.response import Response
+
+auth_info_json = open('auth_info.json', 'r').read()
+login_twitter_json = open('login_twitter.json', 'r').read()
+
+
+def auth_info(request):
+    return Response(auth_info_json)
+
+
+def login_twitter(request):
+    return Response(login_twitter_json)
+
+
+if __name__ == '__main__':
+
+    config = Configurator()
+
+    # configure the routes
+    config.add_route('/auth_info', '/auth_info')
+    config.add_view(auth_info, route_name='/auth_info')
+
+    config.add_route('/login/twitter', '/login/twitter')
```

Comment:
This endpoint should be redirecting as a POST to /login on the calling application with token as a URL parameter, used [here](https://github.com/mediapublic/mediapublic/pull/51/files#diff-0da2e782bbd17fbf39ed578b6d6a4f50R144). 

The real flow is:

1. User goes to /login, hits button to log in with twitter
2. User is sent to `POST /auth/login/twitter` by the form
3. User is redirected to twitter.com's oauth provider, where they log in (or their existing login is used)
3. Twitter redirects them to `POST <ourapp>/login` with the token parameter
4. We grab the auth_info from velruse using the token

The fake flow should be:

1. User goes to /login, hits form button
2. Stub `POST /auth/login/twitter` redirects back to `POST /login` with fake token parameter
3. mediapublic then uses the token to ask /auth_info for info about the login associated with the token
4. The fake login data is sent back